pkg/operator/render/options/manifest.go

```diff
@@ -0,0 +1,95 @@
+package options
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/pflag"
+)
+
+// ManifestOptions contains the values that influence manifest contents.
+type ManifestOptions struct {
+	Namespace             string
+	Image                 string
+	ImagePullPolicy       string
+	ConfigHostPath        string
+	ConfigFileName        string
+	CloudProviderHostPath string
+	SecretsHostPath       string
+}
+
+// NewManifestOptions return default values for ManifestOptions.
+func NewManifestOptions(componentName string) *ManifestOptions {
+	return &ManifestOptions{
+		Namespace:             fmt.Sprintf("openshift-%s", componentName)
+		Image:                 "openshift/origin-hyperkube:latest",
+		ImagePullPolicy:       "IfNotPresent",
```

Comment:
I've found myself switching these to `Always`, now that I'm not building the source locally like I did for cluster up.

Reply:
fixed